torch/fx/experimental/Partitioner.py

```diff
@@ -165,7 +175,7 @@ def partition_graph(
         ret = PartitionResult(dag, module_with_submodules)
         return ret
 
-    def find_single_partition(self, total_size_of_graph) -> None:
+    def find_single_partition(self, total_size_of_graph, logical_device_id) -> None:
```

Comment:
I don't think you need to pass a specific logical ID here, it doesn't necessarily map to the order of device Infos provided. For single partition, the logical device id should just be 0.

Reply:
The reason for this is that for example we have two devices. One device is large enough to hold all nodes but the other is not. In this situation, the larger one is chosen for a single partition and we may need to know which one is the larger one.